src/App.js

```diff
@@ -0,0 +1,41 @@
+/* eslint-disable */
+import './App.css';
+import { Route, Routes, Navigate } from 'react-router-dom';
+import { useSelector } from 'react-redux';
+import Reservations from './components/pages/Reservations';
+import Form from './components/pages/Form';
+import Login from './components/auth/Login'
+import Register from './components/auth/Register'
+import Splash from './components/auth/Splash'
+import Nav from './components/left/Nav';
+import Vehicles from './components/pages/Vehicles';
+import AddVehicle from './components/pages/AddVehicle';
+import VehicleDetails from './components/pages/VehicleDetails';
+
+function App() {
```

Comment:
> Hi @Meltrust,
> 
> We are 4 members, and it says in the requirement that we don't need to implement the delete car functionality but we added the delete reservation from the reservation page. kindly advice.

Look:

![image](https://user-images.githubusercontent.com/57421823/227344692-3f92de82-309c-40d1-9fd1-dabeae8f8b6f.png)

Therefore, this feature is required.

If you are skipping this feature, just add a good explanation in your PR description and concentrate on polishing the other features (please check the delete reservation because it worked janky for me).

When you are done, just submit another review request so we can take a look.

Reply:
hi @Meltrust ,
A user should not be able to delete a vehicle because that should be admin duty. We will be removing the add vehicle feature before deploying so we'll add default rows in the vehicle table that will be populated on the vehicle's page when we deploy.

![image](https://user-images.githubusercontent.com/109461921/227360850-6fc85b2f-0999-48ce-bdef-a7e71adcf58b.png)

Reply:
The screenshot you displayed is meant for the reservation page only